cmd/rancher-desktop-guestagent/main.go

```diff
@@ -68,17 +71,23 @@ func main() {
 	}
 
 	group, ctx := errgroup.WithContext(context.Background())
+
 	if *enableDocker {
+		if *vtunnelAddr == "" {
+			log.Fatal("vtunnel address must be provided when docker is enable.")
+		}
+
 		group.Go(func() error {
-			portTracker := tracker.NewPortTracker()
-			eventMonitor, err := docker.NewEventMonitor()
+			forwarder := forwarder.NewVtunnelForwarder((*vtunnelAddr))
```

Comment:
```suggestion
			forwarder := forwarder.NewVtunnelForwarder(*vtunnelAddr)
```
Unless there was some particular reason for the extra parentheses…

Reply:
I'm surprised gofmt and glalngci-lint didn't catch that 🤔